src/babel-plugin-bare-specifiers.ts

```diff
@@ -34,7 +34,7 @@ type HasSpecifier =
  */
 export const resolveBareSpecifiers = (
     filePath: string,
-    isComponentRequest: boolean,
+    componentRequestRelationship: string,
```

Comment:
I think passing in the packageName would be more inline with what we do in other tools, and we require that rootDir end in packageName. packageName is already available in plasses calling into here.

Reply:
sounds good to me. My intention was not to further couple build with polyserve's specific sibling layout, but if that's already a given then that makes sense as well.